SS_param.tpl

```diff
@@ -278,19 +265,30 @@ PARAMETER_SECTION
   3darray natage_temp(1,pop,1,gmorph,0,nages)
   number ave_age    //  average age of fish in unfished population; used to weight R1
 
-!!//  SS_Label_Info_5.1.3 #Create F parameters and associated arrays and constants
+!!//  SS_Label_Info_5.1.3 #Create M, F, and Z parameters and associated arrays and constants
   init_bounded_number_vector init_F(1,N_init_F,init_F_LO,init_F_HI,init_F_PH)
   matrix est_equ_catch(1,nseas,1,Nfleet)
 
+//  natural, predation and fishing mortality
+  matrix natMparms(1,N_natMparms,1,N_GP*gender)  // will be derived from the MGparms
```

Comment:
question, style: should there be spaces between the indices? I think so...

many lines in this section are also missing `;` at the end of the line

Reply:
Maybe we can discuss this at our next ss3 meeting, since we don't have a set style guide.